src/main/java/frc/robot/OI.java

```diff
@@ -55,69 +61,82 @@ public double getRightJoystick() {
+  // private JoystickButton cargoRocketLevel3 = new JoystickButton(operator2, RobotMap.OI_ROCKET_LEVEL_3_ID);
+
+  // private JoystickButton panelRocketLevel1 = new JoystickButton(operator1, RobotMap.OI_ROCKET_PANEL_LEVEL_1_ID);
+  // private JoystickButton panelRocketLevel2 = new JoystickButton(operator1, RobotMap.OI_ROCKET_PANEL_LEVEL_2_ID);
+  // private JoystickButton panelRocketLevel3 = new JoystickButton(operator1, RobotMap.OI_ROCKET_PANEL_LEVEL_3_ID);
   
-  private JoystickButton cargoShip = new JoystickButton(operator2, RobotMap.OI_CARGO_SHIP_ID);
-  private JoystickButton cargoStation = new JoystickButton(operator2, RobotMap.OI_CARGO_STATION_ID);
+  // private JoystickButton cargoShip = new JoystickButton(operator2, RobotMap.OI_CARGO_SHIP_ID);
+  // private JoystickButton cargoStation = new JoystickButton(operator2, RobotMap.OI_CARGO_STATION_ID);
   
-  private JoystickButton climb1 = new JoystickButton(operator2, RobotMap.OI_CLIMB1_ID);
-  private JoystickButton climb2 = new JoystickButton(operator2, RobotMap.OI_CLIMB2_ID);
-  private JoystickButton climb3 = new JoystickButton(operator2, RobotMap.OI_CLIMB3_ID);
+  // private JoystickButton climb1 = new JoystickButton(operator2, RobotMap.OI_CLIMB1_ID);
+  // private JoystickButton climb2 = new JoystickButton(operator2, RobotMap.OI_CLIMB2_ID);
+  // private JoystickButton climb3 = new JoystickButton(operator2, RobotMap.OI_CLIMB3_ID);
   
-  private JoystickButton clawIn = new JoystickButton(left, RobotMap.OI_CLAW_IN_ID);
-  private JoystickButton clawOut = new JoystickButton(left, RobotMap.OI_CLAW_OUT_ID);
-  private JoystickButton clawGrab = new JoystickButton(right, RobotMap.OI_CLAW_GRAB_ID);
-  private JoystickButton clawRelease = new JoystickButton(right, RobotMap.OI_CLAW_RELEASE_ID);
+  // private JoystickButton clawIn = new JoystickButton(left, RobotMap.OI_CLAW_IN_ID);
+  // private JoystickButton clawOut = new JoystickButton(left, RobotMap.OI_CLAW_OUT_ID);
+  // private JoystickButton clawGrab = new JoystickButton(left, RobotMap.OI_CLAW_GRAB_ID);
+  // private JoystickButton clawRelease = new JoystickButton(right, RobotMap.OI_CLAW_RELEASE_ID);
+
+  // private JoystickButton reverseIntake = new JoystickButton(right, RobotMap.OI_REVERSE_INTAKE_ID);
+  // private JoystickButton placePanel = new JoystickButton(right, RobotMap.OI_PLACE_PANEL_ID);
   
-  private JoystickButton visionAlignment = new JoystickButton(right, RobotMap.OI_VISION_ALIGNMENT_ID);
-  private JoystickButton intakeCargo = new JoystickButton(left, RobotMap.OI_INTAKE_CARGO_ID);
-  private JoystickButton placeCargo = new JoystickButton(right, 2);
+  // private JoystickButton visionAlignment = new JoystickButton(right, RobotMap.OI_VISION_ALIGNMENT_ID);
+  // private JoystickButton intakeCargo = new JoystickButton(left, RobotMap.OI_INTAKE_CARGO_ID);
+  // private JoystickButton placeCargo = new JoystickButton(right, RobotMap.OI_PLACE_CARGO_ID);
```

Comment:
I recommend just removing these comments, if we really want to go back, look back to this PR

Reply:
The comments have been removed.